_posts/2021-04-01-pulp-community-update-march2021.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Pulp Community Update (March 2021)
+author: Melanie Corr
+tags:
+  - newsletter
+---
+
+__If you have any questions or updates you would like to share, we would be delighted to hear from you at `pulp-list@redhat.com`.__
+
+
+## DevConf.cz videos are now available!
+
+While DevConf.cz took place at the end of February, it took a while for the videos from the sessions to be added to YouTube.
+
+If you didn't get a chance, you can watch Pavel and Daniel host a workshop on hosting your own on-premise PyPI.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yxPHEHNJwO4" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Host your own on-premise Ansible Galaxy
+
+Brian also hosted a workshop where he demoed how you can host your very own on-premise Ansible Galaxy. Brian also demoed the power of the new Pulp 3 CLI.
```

Comment:
```suggestion
Brian also hosted a workshop where he demoed how you can host your very own on-premise Ansible Galaxy. Along the way he demostrated the power of the new Pulp 3 CLI.
```

Reply:
I liked this idea but "Along the way" doesn't quite work, imo, so I changed it but not to this. Thank you for the suggestion!